odoo/addons/base/models/ir_sequence.py

```diff
@@ -325,7 +339,7 @@ def _get_number_next_actual(self):
             if seq.sequence_id.implementation != 'standard':
                 seq.number_next_actual = seq.number_next
             else:
-                seq_id = "%03d_%03d" % (seq.sequence_id.id, seq.id)
+                seq_id = "%03d_%03d" % (seq.sequence_id._origin.id, seq._origin.id)
```

Comment:
Do we need this here but not in the `_get_number_next_actual` of `ir.sequence` ?
As there is also an id check?